apps/server/src/controllers/integrationController.ts

```diff
@@ -272,7 +272,8 @@ export function dispatchFromAdapter(
     // WS: {type: 'change', payload: { eventId, property, value } }
     case 'change': {
```

Comment:
The checks we made below only apply for events of type OntimeEvent

Is this what we want or do we want to allow the user to change any property that belongs to any event type?
We need to adapt the logic to reflect this decision and maybe document it with a comment

`// we only allow users to modify events of type of OntimeEvent
`
or

`// users can modify any event type, we need to check if the property is valid`

Reply:
I think option 2 and it almost already happens in the `parse` function in `integrationController.config.ts`